week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/vite.config.ts

```diff
@@ -0,0 +1,12 @@
+import * as path from "path";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  resolve: {
+    alias: {
```

Comment:
To generalnie spoko że stosujesz aliasy, ale moim zdaniem powinny się one zaczynać od jakiegoś charakterystycznego przedrostka np.
`@framework`
albo `src` jako `@`
i wtedy:
`@/framework`


Chodzi o to że jeśli pracowalibyśmy w większym zespole to importy zaczynające się bez ścieżek,
co chwila komuś by się myliły z `node_modules` i faktycznymi libami.

teraz Twoje aliasy sugerują że są `lib`'ami - moim zdaniem to potencjalne źródło nieporozumień.

Reply:
Słuszna uwaga, nie pomyślałem, że może to być mylące z `lib`kami w `node_modules` 🤔

Reply:
Zaktualizowałem aliasy do stylu `@/framework` 👍